support/main/default/triggers/ContactMaster.trigger

```diff
@@ -8,6 +8,6 @@ trigger ContactMaster on Contact(
   before delete
 ) {
   TriggerHandler handler = new TriggerHandler();
-  handler.bind(TriggerHandler.Evt.beforeInsert, new Handler1());
+  //handler.bind(TriggerHandler.Evt.beforeInsert, new Handler1());
```

Comment:
ditto, if we allow for directly binding helpers, we should actually test for it ...

Reply:
Figure we have enough methods to let some directly bind and some auto bind?
Handler2 on account is directly binding